server/pom.xml

```diff
@@ -7,7 +7,14 @@
   <groupId>com.google.sps</groupId>
   <artifactId>server</artifactId>
   <version>1</version>
-  <packaging>war</packaging>
+  <packaging>jar</packaging>
+
+  <!-- <repositories>
```

Comment:
avoid leaving comments in complete PR - do you still need this dependency?

Reply:
No, I've removed the unused dependencies